pom.xml

```diff
@@ -106,12 +107,18 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>branch-api</artifactId>
-            <version>2.0.7</version>
+            <version>2.0.17</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jenkins-ci</groupId>
+                    <artifactId>annotation-indexer</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>cloudbees-folder</artifactId>
-            <version>5.18</version>
+            <version>6.1.2</version>
```

Comment:
Originally had 6.3, which needed a newer core, went back, then updated the core anyway…so could be 6.3. But should not matter.